srcCxx/toc_coredx_dds/ShapeType_configurator_toc_coredx_dds.h

```diff
@@ -58,17 +190,18 @@ class ShapeTypeConfigurator {
                        bool        use_security,
                        const char *governance_uri,
                        const char *permissions_uri,
+                       const char *key_establishment_algo,
                        bool        enable_logging)
   {
     static const char *auth_ca_file  = "file:../TESTONLY_identity_ca_cert.pem";
     static const char *perm_ca_file  = "file:../TESTONLY_permissions_ca_cert.pem";
+#if defined( CDX_ALT_CERT ) && ( CDX_ALT_CERT )
```

Comment:
How are you defining this variables on compilation time? Should they be part of the [makefile](https://github.com/omg-dds/dds-security/blob/master/srcCxx/Makefile.coredx_linux) (or documented in the [README](https://github.com/omg-dds/dds-security/blob/master/README.md)?

Reply:
They are set via the Makefile.coredx_linux [not yet committed], which I didn't include in this PR since it is so vendor specific.  This really was just a quick hack to switch which ID Cert and Permissions file I was using.

That's helpful sometimes, because the ID Cert is a determining factor in deciding which Participant will initiate the Security Handshake.  I like to run the test both ways, to make sure there is no issue with handshakes...

Which, of course, begs the question: should these values be command line parameters?
With defaults that are determined by the vendor 'configurator' code.

If that sounds like a good idea, I can write up a new issue.

Reply:
@ClarkTucker I agree with making this an option!
It may not be super critical, but I think it will be helpful.

Some time ago when testing Diffie-Helmann compatibility I worked on [an ugly hack](https://github.com/omg-dds/dds-security/pull/16/commits/6c4d2ccbdcf3d697dd9b103346807ebe53722e44#diff-23f16dc030fd545d7638b79321d3bc2350353880c2b74323092e6a8a958deb53) to temporally change the subject name of our certificates. As you mention, it is necessary to run the test both ways (each vendor being the initiator one time and the replier in another).

If you can write up a new issue, that would be great. Thanks!